src/ui/InteractiveTutorial/InteractiveTutorialRoot.tsx

```diff
@@ -376,12 +376,12 @@ export function InteractiveTutorialRoot(): React.ReactElement {
             <br />
             To create a new script or edit an existing one, you can use{" "}
           </Typography>
-          <Typography classes={{ root: classes.textfield }}>{"[home ~/]> nano"}</Typography>
+          <Typography classes={{ root: classes.textfield }}>{"[n00dles ~/]> nano"}</Typography>
```

Comment:
wouldnt  these be on home not n00dles though as the previous step says `Let's head home before creating our first script!`

Reply:
Yup thank you for letting me know and reviewing it!